node/src/actors/chain_manager/mod.rs

```diff
@@ -218,6 +218,10 @@ pub struct ChainManager {
     max_reinserted_transactions: usize,
     /// Last received Beacons
     last_received_beacons: Vec<(SocketAddr, Option<LastBeacon>)>,
+    /// a
+    pending_superblock_inventory_msgs: Vec<AddItem>,
+    /// b
+    pending_superblock_notify_msgs: Vec<SuperBlockNotify>,
```

Comment:
This could be a single `Vec<(AddItem, SuperBlockNotify)>`, because we always insert there two messages at the same time, and we always process and clear both vectors at the same time.